scripts/build.js

```diff
@@ -0,0 +1,106 @@
+const { execSync } = require("child_process")
+const { mkdirSync, renameSync, rmSync, existsSync } = require('fs')
+const path = require('path')
+const crypto = require('crypto')
+
+const fs = require('fs')
+const https = require('https')
+
+function execPrint(cmd) {
```

Comment:
I'm thinking it should be clearer if we print `cmd` as well, to let user know what command is being executed.